components/Button.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components"
+
+const StyledButton = styled.a`
```

Comment:
Wanted to bring this up in case anyone else was curious - all the buttons in the design act as links, so this is fine, but if we ever add a button that isn't just a link we should refactor to programmatically render an anchor or button element.

No change needed here, just a note!